### PR TITLE
removed unused dependencies and devDependencies

### LINKED ISSUE #1254: Remove all unused packages from `package.json`
**Is your feature request related to a problem? Please describe.**

1. We have implemented dependabot alerts to let us know when we have outdated packages
2. We have noticed in the past that in some cases the packages listed in `package.json` are no longer being used

**Describe the solution you'd like**
1. Remove all references to unused packages from `package.json`
2. Generate `package-lock.json` appropriately

**Describe alternatives you've considered**
- N/A

**Approach to be followed (optional)**
- N/A

**Additional context**
- N/A

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by Veer0x1**
**What kind of change does this PR introduce?**
Removed unused dependencies

**Issue Number:**

Fixes #1254

**Did you add tests for your changes?**
NA

**Snapshots/Videos:**

**If relevant, did you update the documentation?**

**Summary**
Successfully removed unused dependencies, ran the test, and build, everything is working fine.

**Does this PR introduce a breaking change?**

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,11 +7,11 @@
     "@apollo/client": "^3.4.0-beta.19",
     "@apollo/link-error": "^2.0.0-beta.3",
     "@apollo/react-testing": "^4.0.0",
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
```
Comment (palisadoes) [inline package.json:10]: Why was this added?

**Comment by palisadoes** [comment]
Please fix the failing tests

**Comment by Veer0x1** [comment]
I have added the missing package `postcss-module` and have three failing tests because those tests were not written correctly and merged here(https://github.com/PalisadoesFoundation/talawa-admin/commit/269a3f05a43f7d9473040a8a4b17ca7862685b3b) and also the graphql introspection is failing cause we don't have `UpdateAdvertisment` mutation in API repo, should I create a different PR to fix those test ? And should I add the new `UpdateAdvertisment` mutation in API repo?
Screenshot 2023-12-24 at 1 53 50 PM

**Comment by Cioppolo14** [comment]
@noman2002 @palisadoes Could one of you weigh in on how to proceed?

**Comment by palisadoes** [comment]
@Veer0x1

1. Please fix the tests in this PR
2. Yes open an issue to fix the introspection in Talawa-API.

Thanks for pointing this out. We can now be more vigilant about the introspection check.

**Comment by palisadoes** [comment]
I've also requested clarification here:

- https://github.com/PalisadoesFoundation/talawa-admin/issues/1157#issuecomment-1868997483

**Comment by palisadoes** [comment]
@Veer0x1 Better yet, ask to be assigned to this issue and fix the tests properly as a separate PR

- https://github.com/PalisadoesFoundation/talawa-admin/issues/1157

**Comment by palisadoes** [comment]
This is affecting this PR too
- https://github.com/PalisadoesFoundation/talawa-admin/pull/1076

**Comment by palisadoes** [comment]
The offending PR has been reverted. Please merge with the latest upstream

**Comment by Veer0x1** [comment]
> The offending PR has been reverted. Please merge with the latest upstream

sure

**Comment by Veer0x1** [comment]
> @Veer0x1 Better yet, ask to be assigned to this issue and fix the tests properly as a separate PR
> 
> * [Enabling editing an advertisement #1157](https://github.com/PalisadoesFoundation/talawa-admin/issues/1157)

You have assigned it to vaishali, Let me open a issue in API repo to add the graphql types there so that introspection don't fail

**Comment by palisadoes** [comment]
> > @Veer0x1 Better yet, ask to be assigned to this issue and fix the tests properly as a separate PR
> > 
> > * [Enabling editing an advertisement #1157](https://github.com/PalisadoesFoundation/talawa-admin/issues/1157)
> 
> You have assigned it to vaishali, Let me open a issue in API repo to add the graphql types there so that introspection don't fail

No need. That issue was already created by vaishali

**Comment by Veer0x1** [comment]
> > > @Veer0x1 Better yet, ask to be assigned to this issue and fix the tests properly as a separate PR
> > > 
> > > * [Enabling editing an advertisement #1157](https://github.com/PalisadoesFoundation/talawa-admin/issues/1157)
> > 
> > 
> > You have assigned it to vaishali, Let me open a issue in API repo to add the graphql types there so that introspection don't fail
> 
> No need. That issue was already created by vaishali

will this PR merged when Vaishali fixes those test?

**Comment by palisadoes** [comment]
We reverted the PR. Update with your upstream. The errors should go away

**Comment by Veer0x1** [comment]
@palisadoes I have fetched the latest changes from upstream

**Comment by Veer0x1** [comment]
@noman2002 @beingnoble03 please review this PR.

**Comment by noman2002** [comment]
@Veer0x1 Please fix the conflicting files.

**Comment by Veer0x1** [comment]
@noman2002 there were to many conflicts, so I'm creating a fresh new PR which is in sync with develop branch.
